packages/thumbprint-react/components/Switch/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './index.module.scss';
+
+export const SWITCH_COLOR = {
+    blue: 'blue',
+    green: 'green',
+};
+
+export const SWITCH_SIZES = {
+    small: 'small',
+    large: 'large',
+};
+
+export interface SwitchProps {
+    /**
+     * The `id` is added to the switch as an HTML attribute and passed to the `onChange`
+     * function.
+     */
+    id?: string;
+    /**
+     * Determines if the switch is checked.
+     */
+    isChecked?: boolean;
+    /**
+     * Function that runs when a switch value changes. It receives the new boolean value,
+     * and the provided `id` as such:
+     * `props.onChange(event.target.checked, props.id)`.
+     */
+    onChange: (isChecked: boolean, id: string) => void;
+    /**
+     * Disables the component.
+     */
+    isDisabled?: boolean;
+    /**
+     * Determines how the switch sizes. The default size is `small`.
+     */
+    size?: keyof typeof SWITCH_SIZES;
+    /**
+     * This adds an `aria-label` to the element.
+     * the importance of using labels](https://dequeuniversity.com/rules/axe/4.3/select-name).
+     */
+    accessibilityLabel?: string;
```

Comment:
If this is optional, would we also need to support an `aria-labeledby` attribute? How do we support the presence of an accessible label outside of the component?